robot_tests/scripts/run_regression_test.sh

```diff
@@ -0,0 +1 @@
+robot -i Frontpage -i Searchpage -v browser:headlesschrome -d smear-rahti/robot_tests/regression_test_reports smear-rahti/robot_tests
```

Comment:
Relative path may change once you change the location of dockerfile

Reply:
I think I have to move script files also under the openshift because it might be impossible to:
ADD ../folder/folder with Dockerfile.